apps/chat/src/components/Settings/SettingDialog.tsx

```diff
@@ -107,6 +107,7 @@ export const SettingDialog: FC<Props> = ({ open, onClose }) => {
       dispatch(UIActions.deleteCustomLogo());
     }
 
+    setLocalLogoFile(undefined);
```

Comment:
Why do you set it as `undefined`, not empty string?

Reply:
Because result of the `files.find((file) => file.id === customLogoId);` is `<DialFile | undefined>`.
Then if it not underfined `UIActions.setCustomLogo` action will be dispatched with  `localLogoFile.id`(`<string>`) as value for the custom logo, wich will be saved to the **local store**